### PR TITLE
Add debezium disable schema doc for ticdc

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by asddongmen — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -187,6 +187,10 @@ date-separator = 'day'
 # 注意：该参数只有当下游为存储服务时，才会生效。
 enable-partition-separator = true
 
+# 是否禁用 schema 信息。默认值为 false，即输出 schema 信息。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:190]:
```suggestion
# 是否关闭 schema 信息的输出。默认值为 false，即输出 schema 信息。
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -187,6 +187,10 @@ date-separator = 'day'
...
+# 注意：该参数只有当下游 MQ 且输出协议为 debezium 时才会生效。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:191]:
```suggestion
# 注意：该参数只有当 sink 类型为 MQ 且输出协议为 Debezium 时才生效。
```

File: ticdc/ticdc-changefeed-config.md (hunk 1 above)
Comment (qiancai) [reply ticdc/ticdc-changefeed-config.md:190]: ref: 
image

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**